src/mlpack/core/tree/cover_tree/dual_tree_traverser.hpp

```diff
@@ -87,18 +87,18 @@ class CoverTree<MetricType, StatisticType, MatType, RootPointPolicy>::
    * Helper function for traversal of the two trees.
    */
   void Traverse(CoverTree& queryNode,
-                std::map<int, std::vector<DualCoverTreeMapEntry> >&
+                std::map<int, std::vector<DualCoverTreeMapEntry>, std::greater<int>>&
                     referenceMap);
```

Comment:
```suggestion
                std::map<int,
                         std::vector<DualCoverTreeMapEntry>,
                         std::greater<int>>& referenceMap);
```
Oops, I made this line too long...